src/lib_ncurses.cr

```diff
@@ -68,6 +68,7 @@ lib LibNCurses
   fun notimeout(window : Window, value : Bool)
   fun nodelay(window : Window, value : Bool)
   fun wclear(window : Window)
+  fun wbkgd(window : Window, slot : Int16)
```

Comment:
I suggest to add a method to `Window`, like `Window#background` that takes a color number (or a `ColorPair` object?). This would allow to abstract ncurses internals to the shard users.